config.yaml

```
@@ -522,3 +522,14 @@ Standards:
         Type: Lesson
         UID: d70afaf7a11cea145281fd76ca10f1ab
         Path: /python-oop/problem-set-static-methods-and-class-methods.md
+  -
+    Title: Unit 1 Wrap-up
+    Description: "Reflection Activity"
```

Could this be "Activity: Mind Map," to match the file name, instead?

```suggestion
    Description: "Activity: Mind Map"
```